src/Styleguide/Components/AuctionCard.tsx

```diff
@@ -9,6 +10,36 @@ import {
   Sans,
   Serif,
 } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+
+const zone = time => {
+  return moment(time).tz("America/New_York")
+}
+
+const upcomingLabel = (
```

Comment:
This was taken right from https://github.com/artsy/force/blob/d2d9e3a47f12b4c0ecda98f75e48b86cd9f13665/src/desktop/components/react/auction_block/utils/upcoming_label.js

Reply:
Is this data available on MP?

Reply:
I'm not sure how worth it it is to have every message variation in MP (makes sense if these messages are universal, though, or used elsewhere. That said if the logic is roughly identical to what is currently in MP maybe there's a way to share?